schemas/src/main/java/com/scottlogic/deg/schemas/common/InvalidSchemaException.java

```diff
@@ -0,0 +1,9 @@
+package com.scottlogic.deg.schemas.common;
+
+public class InvalidSchemaException extends Exception {
```

Comment:
Where is the benefit in this exception type over using the `InvalidProfileException` type?

Reply:
The Generator depends on the Schema project, so importing the `InvalidProfileException` caused cyclic dependencies.